apps/search-poc/src/components/ResultTable.tsx

```diff
@@ -25,14 +27,39 @@ const ResultsTable: React.FunctionComponent<ResultTableProps> = ({
   items,
   pageSize = PAGE_SIZE,
   handleClick,
+  handleFileSelect,
+  selectedFileIds
 }) => {
   const [searchValue, setSearchValue] = React.useState();
+  const [selectAll, setSelectAll] = React.useState<boolean>(false);
+  const handelSelectAll = () => {
+    const selectedFiles = !selectAll ?  items.map((item) => { console.log(item); return item.fileId}): [];
+    setSelectAll(!selectAll);
+    console.log(selectedFiles);
+    handleFileSelect &&
+    handleFileSelect(selectedFiles);
+  };
+  const selectHandler = (fileId: string) => {
```

Comment:
Can I ask why you haven't used the built in row selection of the Antd Table component? https://ant.design/components/table/#components-table-demo-row-selection